src/modules/core/transition_composite.c

```diff
@@ -1279,6 +1351,20 @@ static int transition_get_image( mlt_frame a_frame, uint8_t **image, mlt_image_f
 
 			for ( field = 0; field < ( progressive ? 1 : 2 ); field++ )
 			{
+				mlt_slices sliced = NULL;
+
+				// init or set slices obj
+				if ( mlt_properties_get( properties, "sliced_composite" ) )
+				{
+					sliced = mlt_properties_get_data( properties, "sliced_composite_obj", NULL );
+					if ( !sliced )
+					{
+						sliced = mlt_slices_init( 0, SCHED_OTHER, sched_get_priority_max( SCHED_OTHER ) );
```

Comment:
I am OK with leaving this as-is. I just wanted to let you know in case you expected something different. On Another OS, this might really give a higher priority if that is OK.